test/studies/shootout/submitted/fasta3.chpl

```diff
@@ -116,8 +116,8 @@ proc randomMake(desc, a, n) {
   //
   // Add a line of random sequence
   //
-  proc addLine(bytes) {
-    for (r, i) in zip(getRands(bytes), 0..) {
```

Comment:
These edits are a bummer since they'll break our codes that are submitted to the CLBG.  :'(
 
By convention, we usually don't edit tests in this directory, as it's meant to be used to match / track the versions of the benchmarks that are submitted to the shootout website (and serve as a reminder to submit new versions once the new release goes out).  So usually, we'll add a .future, .skipif, or .suppressif to test/studies/shootout/submitted/* tests that start failling.